### PR TITLE
Update 'main' post 24.02

### PR DESCRIPTION
**Pull request by mc-nv**

### CODE REVIEW

File: build.py [hunk 1]
```diff
@@ -72,8 +72,8 @@
 TRITON_VERSION_MAP = {
     "2.44.0dev": (
         "24.03dev",  # triton container
-        "24.01",  # upstream container
-        "1.16.3",  # ORT
+        "24.02",  # upstream container
+        "1.17.2",  # ORT
         "2023.3.0",  # ORT OpenVINO
         "2023.3.0",  # Standalone OpenVINO
```
Comment (jbkyang-nvi) [inline build.py:78]: have we tested that ORT works?

Comment (mc-nv) [reply build.py:78]: Yes, pipeline with this change is in review

File: qa/common/cuda_op_kernel.cu.cc.patch [hunk 2]
```diff
@@ -7,7 +7,7 @@ index a9d66f9..a92e218 100644
 
  #if GOOGLE_CUDA
 -#define EIGEN_USE_GPU
--#include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor"
+-#include "unsupported/Eigen/CXX11/Tensor"  // from @eigen_archive
 -#include "tensorflow/core/util/gpu_kernel_helper.h"
 -#include "tensorflow/core/util/gpu_launch_config.h"
 +//#define EIGEN_USE_GPU
```
Comment (nv-kmcgill53) [inline qa/common/cuda_op_kernel.cu.cc.patch:13]: Why is this getting taken out?

Comment (mc-nv) [reply qa/common/cuda_op_kernel.cu.cc.patch:13]: There was a path change within the TF ontainer

**Comment by mc-nv** [comment]
@jbkyang-nvi , @nv-kmcgill53 I have resolved conversation - could you please continue on review?

**Review by nv-kmcgill53 — APPROVED** [review]

**Pull request merged**